### PR TITLE
Add Change-Theme schematic

### PR DESCRIPTION
**Pull request by mahmut-gundogdu**
Resolves  https://github.com/volosoft/volo/issues/12213 https://github.com/abpframework/abp/issues/14364

### CODE REVIEW

**Review by fatihskilic — APPROVED** [review]

File: npm/ng-packs/packages/schematics/src/commands/create-lib/schema.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     },
     "isSecondaryEntrypoint": {
       "description": "Is secondary entrypoint",
-      "type": "boolean",
+      "type": "boole∂an",
```
Comment (EngincanV) [inline npm/ng-packs/packages/schematics/src/commands/create-lib/schema.json]: might this be wrong?

File: npm/ng-packs/packages/schematics/src/commands/change-theme/schema.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "http://json-schema.org/schema",
+  "$id": "SchematicsABPModuleTemplateCreator",
```
Comment (muhammedaltug) [inline npm/ng-packs/packages/schematics/src/commands/change-theme/schema.json]: please change id

File: npm/ng-packs/packages/schematics/src/commands/change-theme/schema.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  "title": "ABP Theme Style Generator API Schema",
+  "type": "object",
+  "properties": {
+    "name": {
+      "description": "The file extension or preprocessor to use for style files.",
```
Comment (muhammedaltug) [inline npm/ng-packs/packages/schematics/src/commands/change-theme/schema.json]: please change desc

File: npm/ng-packs/packages/schematics/src/commands/change-theme/schema.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+        "items": [
+          { "value": 1, "label": "Basic" },
+          { "value": 2, "label": "Lepton" },
+          { "value": 3, "label": "LeptonXLite" },
+          { "value": 4, "label": "LeptonX" }
+        ]
+      }
+    },
+    "targetProject": {
+      "description": "The name of the project will change the style",
```
Comment (muhammedaltug) [inline npm/ng-packs/packages/schematics/src/commands/change-theme/schema.json]: can you add the description about "project type should be application"

File: npm/ng-packs/packages/schematics/src/commands/change-theme/style-map.ts [hunk 5]
```diff
@@ -0,0 +1,223 @@
...
+    inject: false,
+    bundleName: 'layout-bundle.rtl',
+  },
+  {
+    input: 'node_modules/@abp/ng.theme.lepton-x/assets/css/abp-bundle.rtl.css',
+    inject: false,
+    bundleName: 'abp-bundle.rtl',
+  },
+]);
+// the code written by Github co-pilot. thank go-pilot. You are the best sidekick.
```
Comment (muhammedaltug) [inline npm/ng-packs/packages/schematics/src/commands/change-theme/style-map.ts:222]: i think it should co-pilot instead of go-pilot

Comment (mahmut-gundogdu) [reply npm/ng-packs/packages/schematics/src/commands/change-theme/style-map.ts:222]: Nice catch but no, brand is co-pilot
image

**Review by muhammedaltug — APPROVED** [review]

**Pull request merged**